pmd-java/src/test/kotlin/net/sourceforge/pmd/lang/java/ast/ASTCastExpressionTest.kt

```diff
@@ -140,3 +140,6 @@ class ASTCastExpressionTest : ParserTestSpec({
 
 
 })
+
+val Annotatable.declaredAnnotationsList: List<ASTAnnotation>
```

Comment:
Does this work, if it is declared here in `ASTCastExpressionTest`? I could imagine, this test needs to run before the others... This should probably be moved to the DSL in TestExtensions.kt

Reply:
Toplevel functions and properties in kotlin are translated to static members of a class:
https://kotlinlang.org/docs/reference/java-to-kotlin-interop.html#package-level-functions

So this doesn't need to run. But for clarity, since this is used in several tests, it should indeed be put into TestExtensions.kt